app/assets/javascripts/application.js

```diff
@@ -26,7 +26,7 @@ $(document).ready(function(){
   function routes(url) {
 
     switch (url) {
-      case "#history":
```

Comment:
I'm not sure about removing this hash tag here...

Reply:
On line 39 we're removing it with regex